_posts/2015-03-31-how-we-do-css.md

```diff
@@ -0,0 +1,173 @@
+    |— _fonts.scss
+    |— _icons.scss
+    |— _mixins.scss
+    |— _typography.scss
+    |— _variables.scss
+
+
+#### Toppings
+These can be added to your sandwich to make it as delicious as you want it. These consist of mini components that can be reused throughout the site, but is not necessary for every page.
+Example:
+
+	toppings
+	|— _button.scss
+	|— _input.scss
+	|— _shares.scss
+	|— _tooltip.scss
+
+
+#### Sandwich
+These nom noms can contain a combination of toppings to make a complete sandwich or component. These are usually a bit more complex and a little more specific. Sandwiches can be used throughout the entire site and can be customized per section of the site to meet that section's needs or requirements.
+Example:
+
+	sandwich
+	|— _accordion.scss
+	|— _featured.scss
+	|— _form.scss
+	|— _hero-slider.scss
+	|— _modal.scss
+	|— _pagination.scss
+	|— _rangeslider.scss
+
+
+#### Example of our overall structure:
+
+	assets-build/css-blt
+	|— checkout/
+	|— classes/
+	|— consumer/
+	|— header-footer/
+	|— plate/
+	|— sandwiches/
+	|— shop/
+	|— toppings/
+	|— _settings.scss
+	|— classes.scss
+	|— consumer.scss
+	|— header-footer.scss
+	|— main.scss
+	|— shop.scss
+
+
+#### main.scss - Consists of the basics of our pages’ layout.
+
+<script src="https://gist.github.com/neptunz/0d6bfe832eb85b6e73a7.js"></script>
+
+
+### CSS structure
+This is an example of how we expect our stylesheets to be laid out. This is not an actual file.
+
+<script src="https://gist.github.com/neptunz/6b55d2e0db2d0190c540.js"></script>
+
+
+#### + Each file is named after the page or the page module it supports
+#### + We never style ID’s only Classes
+#### + Keep nesting down to a minimum and no deeper than 4 levels including pseudo selectors
+#### + Group related properties wherever possible
+#### + Media queries go after properties and the order should reflect mobile first
+
+
+### Naming
+
+
+#### Variables and Mixins
+Our variables and mixing naming is pretty specific in order to avoid any confusion. We have been here long enough to understand that the company likes to make simple to drastic changes to support the business in general. We have to develop keeping scalability in mind and this variable/mixin strategy just makes more sense for us.
+
+
+#### Mixin
+
+<script src="https://gist.github.com/neptunz/65b4c6cc19a4c4ad2fd6.js"></script>
+
+
+#### Color Variables
+
+<script src="https://gist.github.com/neptunz/c473d1fb7056fb22ef46.js"></script>
+
+
+#### Classes
+Our preferred technique is a more descriptive naming convention over deep nesting.
+
+<script src="https://gist.github.com/neptunz/2c49168fb55be1cbbd25.js"></script>
+
+
+#### Media Queries
+For the most part we target based on viewport width. We have a feature or two that depends on the detection of touch. We stick to three basic breakpoints and rarely make exceptions for custom breakpoints. We don’t want to compromise code quality to consider every viewport width consideration.
```

Comment:
For the most part<strong>,</strong> we target based on viewport width. 

You should use a comma if an introductory statement like this four words or more.